storage/aws/aws.go

```diff
@@ -59,9 +59,10 @@ import (
 )
 
 const (
-	entryBundleSize = 256
-	logContType     = "application/octet-stream"
-	ckptContType    = "text/plain; charset=utf-8"
+	entryBundleSize       = 256
+	logContType           = "application/octet-stream"
+	ckptContType          = "text/plain; charset=utf-8"
+	minCheckpointInterval = 1 * time.Second
```

Comment:
```suggestion
	minCheckpointInterval = time.Second
```
Remove `1 * ` to align with the rest of the storage.